sql/backup.sh

```diff
@@ -184,7 +184,20 @@ pg_rman show detail -B ${BACKUP_PATH} > ${TEST_BASE}/TEST-0011.log 2>&1
 grep OK ${TEST_BASE}/TEST-0011.log | grep FULL | wc -l
 grep ERROR ${TEST_BASE}/TEST-0011.log | grep ARCH | wc -l
 
-echo '###### BACKUP COMMAND TEST-0009 ######'
+echo '###### BACKUP COMMAND TEST-0009 #######'
+echo '###### confirm incremental backup is right #######'
+init_catalog
+pg_rman backup -B ${BACKUP_PATH} -b full -p ${TEST_PGPORT} -d postgres --quiet;echo $?
+pg_rman validate -B ${BACKUP_PATH} --quiet
+pgbench -i -s 50 -d postgres >  ${TEST_BASE}/pgbench.log 2>&1
+pg_rman backup -B ${BACKUP_PATH} -b incremental -p ${TEST_PGPORT} -d postgres --quiet;echo $?
+pg_rman validate -B ${BACKUP_PATH} --quiet
+pg_rman backup -B ${BACKUP_PATH} -b incremental -p ${TEST_PGPORT} -d postgres --quiet;echo $?
+pg_rman validate -B ${BACKUP_PATH} --quiet
+pg_rman show detail -B ${BACKUP_PATH} > ${TEST_BASE}/TEST-0012.log 2>&1
+grep -c 16kB ${TEST_BASE}/TEST-0012.log
```

Comment:
Because the size of the data can be bigger or smaller in the future, is it better to check with a certain range of data size? For example, check if more than 20kB and less than 30kB.

Reply:
In many environments, I confirmed that the acquired data size is 16kB when there is no update.
And I cannot find the command to check the number is more than 20kB and less than 30kB.

Reply:
It's OK since this  seems to be related to pg_rman's code.